resources/js/views/Recipes.vue

```diff
@@ -160,6 +163,24 @@
         this.editedIndex = -1;
       }, 300);
     },
+
+    download() {
+        if (this.editedIndex > -1) {
+            Object.assign(this.recipes[this.editedIndex], this.editedItem);
+            this.snackbarText = "Recipe Downloaded";
+            this.snackbar = true;
+        
+          var vm= this,
```

Comment:
the use of var is deprecated change to let or const

Reply:
what is ```vm```, name your variables better when you do create them itll save you a headache later on.

Reply:
VM is just nothing, its just a variable I declared to replace "this" -- for some reason BLOB is gimmicky and when I used this instead of vm it didn't work but when I used vm declared as this, it worked. It needs to stay there and has no other use but that